### PR TITLE
Drop-down Editors: add a note to Getting Started tutorials

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

**Review by AnnaKondratova — APPROVED** [review]

File: concepts/05 UI Components/Lookup/00 Getting Started with Lookup/00 Getting Started with Lookup.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#include uiwidgets-editors-choose-a-dropdown-editor
+
 #include tutorials-intro-installationnote
```
Comment (albertov05) [inline concepts/05 UI Components/Lookup/00 Getting Started with Lookup/00 Getting Started with Lookup.md:4]: Can you change the following line below?

Line 9:
`Each section in this tutorial covers a single configuration step.`

Can we change "covers" to "describes"?

File: concepts/05 UI Components/SelectBox/00 Getting Started with SelectBox/00 Getting Started with SelectBox.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+#include uiwidgets-editors-choose-a-dropdown-editor
+
 #include tutorials-intro-installationnote
```
Comment (albertov05) [inline concepts/05 UI Components/SelectBox/00 Getting Started with SelectBox/00 Getting Started with SelectBox.md:4]: Line 9 (below):
`Each section in this tutorial covers a single configuration step.`

Can we change "covers" to "describes"?

File: concepts/05 UI Components/TagBox/00 Getting Started with TagBox/00 Getting Started with TagBox.md [hunk 3]
```diff
@@ -1,3 +1,5 @@
+#include uiwidgets-editors-choose-a-dropdown-editor
+
 #include tutorials-intro-installationnote
```
Comment (albertov05) [inline concepts/05 UI Components/TagBox/00 Getting Started with TagBox/00 Getting Started with TagBox.md:4]: Line 9 (below):
`Each section in this tutorial covers a single configuration step.`

See my comment above regarding "covers".

File: includes/uiwidgets-editors-choose-a-dropdown-editor.md [hunk 4]
```diff
@@ -0,0 +1 @@
+Drop-down editors allow users to navigate through a list of items, select one or multiple items, search through the list. To choose between DevExtreme drop-down editors and learn more about the component's features, refer to the following article: [How to Choose a Drop-Down Editor](/Documentation/Guide/Common/Editors_Concepts/How_to_Choose_a_Drop-Down_Editor/).
```
Comment (albertov05) [inline includes/uiwidgets-editors-choose-a-dropdown-editor.md]:
```suggestion
Drop-down editors allow users to navigate through a list of items, select one or multiple items, and search through the list. To learn how to choose a DevExtreme drop-down editor and for more details about the component's features, refer to the following article: [How to Choose a Drop-Down Editor](/Documentation/Guide/Common/Editors_Concepts/How_to_Choose_a_Drop-Down_Editor/).
```

File: concepts/05 UI Components/Lookup/00 Getting Started with Lookup/00 Getting Started with Lookup.md (hunk 1 above)
Comment (vladaskorohodova) [reply concepts/05 UI Components/Lookup/00 Getting Started with Lookup/00 Getting Started with Lookup.md:4]: Noted

File: concepts/05 UI Components/SelectBox/00 Getting Started with SelectBox/00 Getting Started with SelectBox.md (hunk 2 above)
Comment (vladaskorohodova) [reply concepts/05 UI Components/SelectBox/00 Getting Started with SelectBox/00 Getting Started with SelectBox.md:4]: Noted

File: concepts/05 UI Components/TagBox/00 Getting Started with TagBox/00 Getting Started with TagBox.md (hunk 3 above)
Comment (vladaskorohodova) [reply concepts/05 UI Components/TagBox/00 Getting Started with TagBox/00 Getting Started with TagBox.md:4]: Noted

**Pull request merged**